### ISSUE TITLE
Windows desktop app with ReorderableListView: Failed to update ui::AXTree

### ISSUE THREAD

**Issue by sharpsaravanan** [flutter/flutter#118221]
Windows desktop app with ReorderableListView causes 

[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 20 is already pending for creation, cannot be a new child

 <details>
<summary>flutter doctor -v</summary>

```
[√] Flutter (Channel stable, 3.3.10, on Microsoft Windows [Version 10.0.22000.1335], locale en-IN)
    • Flutter version 3.3.10 on channel stable at C:\Users\sharp\Documents\flutter_windows_2.5.3-stable\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 135454af32 (4 weeks ago), 2022-12-15 07:36:55 -0800
    • Engine revision 3316dd8728
    • Dart version 2.18.6
    • DevTools version 2.15.0

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at C:\Users\sharp\AppData\Local\Android\sdk
    • Platform android-32, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Enterprise 2019 16.8.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
    • Visual Studio Enterprise 2019 version 16.8.30804.86
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 2021.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[√] VS Code (version 1.74.2)
    • VS Code at C:\Users\sharp\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.56.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.1335]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 108.0.5359.125
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 108.0.1462.76

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by danagbemava-nc** [comment]
Hi @sharpsaravanan, please provide a complete minimal reproducible code sample along with steps to reproduce the issue. 

Please also provide the full crash log and the full output of `flutter run -v`. 

Thank you

**Comment by sharpsaravanan** [comment]
[main.txt](https://github.com/flutter/flutter/files/10393524/main.txt)
[output of flutter run -v..txt](https://github.com/flutter/flutter/files/10393569/output.of.flutter.run.-v.txt)

**Comment by danagbemava-nc** [comment]
Hi @sharpsaravanan, it looks like this was reported in https://github.com/flutter/flutter/issues/98099 and has since been fixed. Can you check on the master channel to see if this still reproduces there?

You can switch to the master channel by running 

```
flutter channel master
flutter upgrade
```

If this still reproduces on the `master` channel, please provide the output of `flutter doctor -v` on the master channel. 

Thank you

**Comment by sharpsaravanan** [comment]
Thank you, I am using Channel stable
[flutter doctor -v.txt](https://github.com/flutter/flutter/files/10403183/flutter.doctor.-v.txt)
